documentation/jetty/modules/programming-guide/pages/server/http.adoc

```diff
@@ -1286,23 +1286,23 @@ include::code:example$src/main/java/org/eclipse/jetty/docs/programming/server/ht
 [[handler-use-webapp-context-class-loading]]
 .WebAppContext Class Loading
 
-The Servlet specification requires that a web application class loader must load the web application classes from `WEB-INF/classes` and `WEB_INF/lib`.
-The web application class loader is special because it behaves differently from typical class loaders: where typical class loaders first delegate to their parent class loader and then try to find the class locally, the web application class loader first tries to find the class locally and then delegates to the parent class loader.
-The typical class loading model, parent-first, is _inverted_ for web application class loaders, as they use a child-first model.
+The Servlet specification requires that a web application class loader must load classes found in the web application `WEB-INF/classes` and `WEB_INF/lib` in preference to using the parent class loader.
+This is significantly different to typical java class loaders: where loaders first delegate to their parent class loader.
+For Servlet specification compliant web applications, the typical class loading model, parent-first, is _inverted_.
 
-Furthermore, the Servlet specification requires that web applications cannot load or otherwise access the Servlet container implementation classes, also called _server classes_.
+Furthermore, the Servlet specification requires that web applications cannot load or otherwise access the Servlet container implementation classes, also called _hidden classes_ (previously _server classes).
```

Comment:
```suggestion
Furthermore, the Servlet specification requires that web applications cannot load or otherwise access the Servlet container implementation classes, also called _hidden classes_ (previously called _server classes_).
```